spec/models/review_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
+
+RSpec.describe Review, type: :model do
+  describe 'relationships' do
+    it {should belong_to :shelter}
+  end
+
+  describe "validations" do
+    it { should validate_presence_of :title
+      should validate_presence_of :rating
+      should validate_presence_of :content
+      should validate_presence_of :shelter
+      should validate_presence_of :picture}
```

Comment:
Is a picture required? App broke a couple of times when I was playing in the server because either the picture path wasn't perfect or there were no quotes, or ... Feel like this is going to cause issues down the line but I'm not sure.

Reply:
I think it might be. I removed it because it was the only way I could figure out how to make the picture optional, but Megan mentioned something about an optional validation within the model, so I will take a look at that as well.